src/main/java/life/genny/messages/managers/QSendGridMessageManager.java

```diff
@@ -68,7 +69,19 @@ public void sendMessage(BaseEntityUtils beUtils, BaseEntity templateBe, Map<Stri
 				log.info("attributeCode=" + ea.getAttributeCode() + ", value=" + ea.getObjectAsString());
 			}
 
-			String recipient = recipientBe.getValue("PRI_EMAIL", null);
+			String recipient = null;
+			// send email to secondary email if it present.
+			if(recipientBe != null)
+			{
+				String additionalEmail = recipientBe.getValue("PRI_EMAIL_ADDITIONAL", null);
+				String primaryEmail = recipientBe.getValue("PRI_EMAIL", null);
```

Comment:
You may want to avoid checking for primary unless secondary is null. Just is a good habit to get into 😄

Reply:
resolved with function